specs/glossary.md

```diff
@@ -139,11 +140,20 @@ Different transaction types can contain different payloads, and be handled diffe
 The fork choice rule is the rule used to determined which block is to be considered as the head of a blockchain. On L1,
 this is determined by the proof of stake rules.
 
-L2 also has a fork choice rule, although the rules vary depending on wether we want the sequencer-confirmed head, the
+L2 also has a fork choice rule, although the rules vary depending on whether we want the sequencer-confirmed head, the
 on-chain-confirmed head, or the on-chain-finalized head.
 
 > TODO: define and link to those concepts
 
+## Priority Gas Auction
+
+Transactions are in ethereum are ordered by the price that the transaction pays to the miner. Priority Gas Auctions
+(PGAs) occur when multiple parties are competing to be the first transaction in a block. Each party continuously
+updates the gas price of their transaction. PGAs occur when there is value in submitting a transaction before other
+parties (like being the first deposit or submitting a deposit before there is not more guaranteed gas remaining).
+PGAs tend to have negative externalities on the network due to a large amount of transactions being submitted in a
+very short amount of time.
```

Comment:
I don't think it's a great explanation of the negative externalities but I don't think it's really our responsibility to explain them as part of this doc. Perhaps flashbots has a blog post someplace but no need to update this

Reply:
Agreed, that an external link could serve as alternative. Though I remember there was sometimes pushback for including external links.

Assuming this is kept, then:
> are competing to be the first transaction in a block. 

Not necessarily, could be for inclusion in the block at all, for instance.

> PGAs tend to have negative externalities on the network due to a large amount of transactions being submitted in a
very short amount of time.

I don't think that's accurate. In fact it's rather the reverse: a lot of transaction being submitted (e.g. for an NFT mint) create a PGA to be included in the block.

The negative consequence is more about normal people being unable to transact for an extended period of time.
What EIP-1559 achieves is temporarily extended throughput by raising the cost (which PGA participants are willing to bear anyway) so that the period of congestion due to the PGA subsides faster.

Reply:
They talk about it a bit, but don't have a good reference that I could easily find.